MatrixSDK/Data/MXRoom.m

```diff
@@ -2158,46 +2158,37 @@ - (void)getReplyContentBodiesWithEventToReply:(MXEvent*)eventToReply
                     replyContentFormattedBody:(NSString**)replyContentFormattedBody
                               stringLocalizer:(id<MXSendReplyEventStringLocalizerProtocol>)stringLocalizer
 {
-    if (eventToReply.eventType == MXEventTypePollStart)
-    {
-        NSString *question = [MXEventContentPollStart modelFromJSON:eventToReply.content].question;
-
-        *replyContentBody = [self replyMessageBodyFromSender:eventToReply.sender
-                                           senderMessageBody:question
-                                      isSenderMessageAnEmote:NO
-                                     isSenderMessageAReplyTo:eventToReply.isReplyEvent
-                                                replyMessage:textMessage];
-        
-        // As formatted body is mandatory for a reply message, use non formatted to build it
-        NSString *finalFormattedTextMessage = formattedTextMessage ?: textMessage;
-        
-        *replyContentFormattedBody = [self replyMessageFormattedBodyFromEventToReply:eventToReply
-                                                          senderMessageFormattedBody:question
-                                                              isSenderMessageAnEmote:NO
-                                                               replyFormattedMessage:finalFormattedTextMessage
-                                                                     stringLocalizer:stringLocalizer];
-        return;
-    }
-    
     NSString *msgtype;
     MXJSONModelSetString(msgtype, eventToReply.content[kMXMessageTypeKey]);
     
-    if (!msgtype)
-    {
-        return;
-    }
-    
     BOOL eventToReplyIsAlreadyAReply = eventToReply.isReplyEvent;
     BOOL isSenderMessageAnEmote = [msgtype isEqualToString:kMXMessageTypeEmote];
     
     NSString *senderMessageBody;
     NSString *senderMessageFormattedBody;
     
-    if (eventToReply.location)
+    if (eventToReply.eventType == MXEventTypePollStart)
+    {
+        NSString *question = [MXEventContentPollStart modelFromJSON:eventToReply.content].question;
+        
+        senderMessageBody = question;
+        senderMessageFormattedBody = senderMessageBody;
```

Comment:
If this happen on several other cases, I suppose we could add a specific setter that updates both

Reply:
Good catch I simplified the code